deploy/kubernetes/console/templates/psp.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if and .Values.console.pspEnabled (not .Values.console.pspName) }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ .Release.Name }}-psp
+{{- if .Values.console.pspAnnotations }}
+  annotations:
+{{ toYaml .Values.console.pspAnnotations | indent 4 }}
```

Comment:
Should console.pspAnnotations and console.pspExtraLabels be documented in the README?

Reply:
Yes - done